app/search/classification/domain.py

```diff
@@ -0,0 +1,23 @@
+from urllib.parse import urlparse
+
+from .base import BaseClassifier
+
+
+class DomainClassifier(BaseClassifier):
+    """
+    Classifier that is applied if the returned result is the root address of a
+    domain name. Adds:
+
+    fqdn - the fully qualified domain name
+    logo - the URL to a logo, as determined by Clearbit.
```

Comment:
I think that's probably right. We should definitely have some documented standards around how they should behave, because they are critical.